library/Freckle/App/Database.hs

```diff
@@ -22,11 +22,11 @@ module Freckle.App.Database
   , envPostgresPasswordSource
   ) where
 
-import Prelude
+import Freckle.App.Prelude
+import Prelude (read)
```

Comment:
When doing something like this (using a safe prelude and an unsafe function from `Prelude`). My convention is to `import Prelude as Unsafe (read)`. Then it's nice and clear that you're using `Unsafe.read` later. I also don't consider the `Unsafe` import a "prelude", so I group it down with the rest.

Reply:
I like that -- it makes things pretty clear.